homeassistant/components/irm_kmi/entity.py

```diff
@@ -0,0 +1,32 @@
+"""Base class shared among IRM KMI entities."""
+
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, IRM_KMI_NAME
+from .utils import preferred_language
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class IrmKmiBaseEntity(CoordinatorEntity):
+    """Base methods for IRM KMI entities."""
+
+    _attr_attribution = (
+        "Weather data from the Royal Meteorological Institute of Belgium meteo.be"
+    )
+
+    def __init__(self, entry: ConfigEntry, name: str) -> None:
```

Comment:
use the typed config entry

Reply:
Fixed in 31e15bbdb4094e9403aae8ac4d042de5038b6283